Test/E2ETests/TestApps/Net452/E2ETestApp/ApplicationInsights.config

```diff
@@ -36,6 +36,7 @@
         <Add>e2etests_azureemulator_1</Add>
       </ExcludeComponentCorrelationHttpHeadersOnDomains>
       <IncludeDiagnosticSourceActivities>
+        <Add>Microsoft.Azure.EventHubs</Add>
```

Comment:
Do you plan to add some functional tests for ServiceBus / EventHub? I see that configuration of the tests will now force them to listen to those dependencies but we do not produce such calls in the test web apps currently to validate their collection.

Reply:
If these are expected than we will add them, although I don't think we'll be able to do that for Beta. We can make sure they are in place before Stable though

Reply:
We typically add functional tests coverage for the auto-collected dependencies and requests to make sure they are flowing in the actual web application before we ship the feature as well as for the regression detection purposes. I would recommend adding them in the stable timeframe if you already had any other type of the validation for beta 2 other than unit tests (e.g. manual test). Let's discuss more offline if needed.

Reply:
Let's sync up on the details next week - currently the corresponding changes in EH & SB SDKs have not been published yet so we'd have to emulate them anyway. I've created an issue for tracking this: #758